File: apps/HSMS-fe/src/app/models/City.dto.ts
export interface CityMastersDto {
  id: number;
  stateId: number;
  stateName: string;
  name: string;
  postalCode: string;
  createdAt: string;
  updatedAt: string;
}


export interface CreateCityDto {
   stateId: number;
  name: string;
  postalCode: string;
}

File: libs/contracts/Dto/CityDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HSMS.contracts.Dto
{
    public class CityDto
    {
        public int Id { get; set; }

        public int StateId { get; set; }

        public string StateName { get; set; } = string.Empty;

        public string Name { get; set; } = string.Empty;

        public string? PostalCode { get; set; }

        public DateTimeOffset CreatedAt { get; set; }

        public DateTimeOffset? UpdatedAt { get; set; }
    }
}
